### DISCUSSION TITLE
Include both import parameter and table parameter in conn.call

### DISCUSSION THREAD

**Discussion by Lemurboy** [SAP/PyRFC#261] [Q&A] [answered]
Hello,

Can anyone help me with how I should call a function module that takes both an import parameter and table parameters? By table parameters, I mean that in addition to the import parameter, the function module has a table entry that needs to have its fields filled out. I've tried passing two the import and table parameters separately as kwargs in the conn.call method, combining them into one dictionary, and making the import parameter as a dictionary and table parameter as a list of dictionaries but they all lead to RFC_INVALID_PARAMETER error because the program thinks the table parameters are a part of the import parameters.

Here's how my call method is at the moment:
import_param = { 'abc': 'def' }
table_param = [{ 'qwe': 'rty' }]
result = conn.call('Function_Name', **import_param, **table_param[0])
//import_param is recognized, but program can't separate the table_param and thinks it is a part of the import_param

**Comment by bsrdjan** [comment, marked as answer]
Table parameters are handled the same way like variable and structure parameters. You can try the `STFC_STRUCTURE` example from README first:
https://github.com/SAP/PyRFC#call-abap-function-modules

**Reply by Lemurboy** [reply]
Thank you for your reply. I saw this example before and didn't understand it fully back then. I just realized that the example's 'RFCTABLE' parameter is the actual name of the table parameter in the SAP, so I changed my parameter name accordingly and it is working like a charm. Thank you again for your time!
